### PR TITLE
convenience launch files for joint_limited versions

### PR DESCRIPTION
**Pull request by fmessmer**
solves ros-industrial/universal_robot/issues/50

### CODE REVIEW

File: ur_bringup/ur10_joint_limited.launch [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<!--
+  Universal robot ur10 launch.  Wraps ur10_bringup.launch
+  
+  NOTE: THIS FILE IS DEPRECATED AND MAY BE REMOVED IN THE FUTURE
+
```
Comment (gavanderhoorn) [inline ur_bringup/ur10_joint_limited.launch:6]: Should we perhaps take this chance to remove the deprecated launchfiles? I often have a hard enough time figuring out which launchfile I should start now, let alone when we add an additional few.

**Comment by gavanderhoorn** [comment]
+1 from me. This would fix #50.

**Comment by fmessmer** [comment]
An option for solving this would be to rename launch files that should not be called directly to launchfilename.xml
However, we would than have a problem with backwards compatibility again...

**Comment by gavanderhoorn** [comment]
ipa-fxm wrote:

> An option for solving this would be to rename launch files that should not be called directly to <launchfilename>.xml
> However, we would than have a problem with backwards compatibility again...

we try to adopt a tick-tock approach with ROS-I: deprecate in one
release (Groovy), remove in the following (Hydro). AFAIK the deprecation
warning has been there in Groovy, which would give us the option to
remove it in Hydro.

**Comment by shaun-edwards** [comment]
Can this be merged or do we want to move deprecated launch files?

**Comment by gavanderhoorn** [comment]
The current changes are ok. So this can be merged.

Perhaps the removal of deprecated files should be done in a separate PR.

But I do vote for removing the files.

**Pull request merged**